docs/manual/installation.rst

```diff
@@ -13,9 +13,16 @@ instructions on `the composer website <https://getcomposer.org/download/>`_ if
 you don't have it installed yet.
 
 N.b.: You will need to ensure that your Composer ``autoload`` settings are
-correct. PHPSpec will not be able to detect classes, even ones it has created,
+correct. phpspec will not be able to detect classes, even ones it has created,
 unless this is working. This is a common issue which causes confusion when
-installing PHPSpec.
+installing phpspec.
+
+The ``autoload`` section of your ``composer.json`` file may look something like
+this:
+
+.. code-block:: json
+
+    "autoload": {"psr-0": {"": "src"}}
```

Comment:
This would be better `{"autoload": {"psr-0": {"": "src/"}}}` (and that slash is important)

Pretty print would be even better:

```json
{
  "autoload": {
    "psr-0": {
      "": "src/"
    }
  }
}
```